webpack/components/PreupgradeReportsList/components/helpers.js

```diff
@@ -1,28 +1,157 @@
+        <Icon type="pf" name="warning-triangle-o" /> Has Remediation
+      </ListView.InfoItem>
+    );
+  }
+
+  return <EmptyInfoItem key={key} entry={entry} attr="remediation" />;
+};
+
+export const getTitle = entry => {
+  if (entry.title) {
+    return (
+      <Grid.Row>
+        <Grid.Col md={1}>
+          <strong>Title</strong>
+        </Grid.Col>
+        <Grid.Col md={8}>{entry.title}</Grid.Col>
+      </Grid.Row>
+    );
+  }
+  return <div />;
+};
+
+export const getSeverity = entry => {
+  if (entry.severity) {
+    return (
+      <Grid.Row className="top-padded">
+        <Grid.Col md={1}>
+          <strong>Risk Factor</strong>
+        </Grid.Col>
+        <Grid.Col md={8}>
+          <Icon
+            type="pf"
+            name="resources-full"
+            style={{ color: presentSeverity(entry.severity) }}
+          />{' '}
+          {entry.severity}
+        </Grid.Col>
+      </Grid.Row>
+    );
+  }
+  return <div />;
+};
+
+export const getSummary = entry => {
+  if (entry.summary) {
+    return (
+      <Grid.Row className="top-padded">
+        <Grid.Col md={1}>
+          <strong>Summary</strong>
+        </Grid.Col>
+        <Grid.Col md={8} className="pre-wrap">
+          {entry.summary}
+        </Grid.Col>
+      </Grid.Row>
+    );
+  }
+  return <div />;
+};
+
+export const getTags = entry => {
+  if (entry.tags) {
+    return (
+      <Grid.Row className="top-padded">
+        <Grid.Col md={1}>
+          <strong>Tags</strong>
+        </Grid.Col>
+        <Grid.Col md={8}>{tagInfo(entry.tags)}</Grid.Col>
+      </Grid.Row>
+    );
+  }
+  return <div />;
+};
+
+export const getExternals = entry => {
+  if (entry.detail && entry.detail.external) {
+    return entry.detail.external.map(attr => {
+      const key = itemIteratorId(entry, attr);
+      return (
+        <Grid.Row className="top-padded" key={key}>
+          <Grid.Col md={1}>
+            <strong>Links</strong>
+          </Grid.Col>
+          <Grid.Col md={8}>
+            <a href={attr.url}>{attr.title}</a>
+          </Grid.Col>
+        </Grid.Row>
+      );
+    });
+  }
+  return <div />;
+};
+
+export const getRemediations = entry => {
+  if (entry.detail && entry.detail.remediations) {
+    return entry.detail.remediations.map(attr => {
+      const key = itemIteratorId(entry, attr);
+      if (attr.type === 'hint') {
+        return (
+          <Grid.Row className="top-padded" key={key}>
+            <Grid.Col md={1}>
+              <strong>Hint</strong>
+            </Grid.Col>
+            <Grid.Col md={8}>{attr.context}</Grid.Col>
+          </Grid.Row>
+        );
+      }
+      if (attr.type === 'command') {
+        return (
+          <Grid.Row className="top-padded" key={key}>
+            <Grid.Col md={1}>
+              <strong>Command</strong>
+            </Grid.Col>
+            <Grid.Col md={8}>
+              <code>{attr.context.join(' ')}</code>{' '}
+              <Button bsSize="xsmall">Run Remediation</Button>
```

Comment:
No need for the button, we will not be running each remediation separately. #24 and #32 will take care of it.